src/bun.js/api/server.zig

```diff
@@ -183,6 +249,34 @@ pub const ServerConfig = struct {
                     any = true;
                 }
             }
+
+            if (obj.getTruthy(global, "key")) |js_obj| {
+                if (js_obj.jsType().isArray()) {
```

Comment:
can we throw an error in `else` here to say that it is not valid?

Reply:
"Expected `key` to be an array of strings`

Reply:
actually I think we should do

`JSC.Node.StringOrBuffer` for the element type in these

That lets us accept:
- string
- any typed array view
- arraybuffer

technically we should also support Bun.file() for these, but we can leave that for another day